### PR TITLE
Fix Clipboard Button in Run Details

### PR DESCRIPTION
**Pull request by THWiseman**
Fixed an issue with a clipboard button in the Workflow Details page where it couldn't receive click events. [WX-1063]

Testing I did:
- Verified that the collapse menu still opens and closes when clicking anywhere (except on the clipboard button!) on the line with the title. 
- Verified that the clipboard button is visually in the right spot and does indeed copy stuff to your clipboard. 

https://broadworkbench.atlassian.net/browse/WX-1063?atlOrigin=eyJpIjoiY2NkNzMwNzBiNmM0NGJmODk3OWRkMTZhZTlmNDgyYzgiLCJwIjoiaiJ9

[WX-1063]: https://broadworkbench.atlassian.net/browse/WX-1063?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jgainerdewar — COMMENTED** [review]
Hey presto, look at that!

We have compliance requirements around having every change associated with a Jira ticket - would you mind creating a ticket for this bug and linking to the PR?

**Comment by THWiseman** [comment]
> Hey presto, look at that!
> 
> We have compliance requirements around having every change associated with a Jira ticket - would you mind creating a ticket for this bug and linking to the PR?

All set! See WX-1063. Updated description above.

**Review by aednichols — APPROVED** [review]

File: src/pages/RunDetails.js [hunk 1]
```diff
@@ -210,14 +210,12 @@ export const RunDetails = ({ submissionId, workflowId }) => {
...
-                        text: JSON.stringify(failures, null, 2),
-                        style: { marginLeft: '0.5rem' },
-                        onClick: e => e.stopPropagation() // this stops the collapse when copying
-                      })
-                    ])
+                    title: div({ style: elements.sectionHeader }, 'Workflow-Level Failures'),
+                    afterTitle: h(ClipboardButton, {
+                      text: JSON.stringify(failures, null, 2),
+                      style: { marginLeft: '0.5rem' },
+                      onClick: e => e.stopPropagation() // this stops the collapse when copying
```
Comment (JVThomas) [inline src/pages/RunDetails.js:217]: Didn't expect our conversation on event propagation to be useful so soon

Comment (JVThomas) [reply src/pages/RunDetails.js:217]: One suggestion: Can you add the `e.stopPropagation` call on the `onClick` method in `ClipboardButton` itself? The component should be defined on `common.js`.

If the method call is defined there then you wouldn't have to add the onClick every time you used `ClipboardButton` on the template

**Review by JVThomas — APPROVED** [review]

**Pull request merged**